ci-operator/step-registry/telcov10n/functional/cnf-ran/cnf-tests-sno-worker/telcov10n-functional-cnf-ran-cnf-tests-sno-worker-commands.sh

```
@@ -0,0 +1,214 @@
+}
+trap cleanup EXIT
+
+echo "Extract spoke kubeconfig from hub via bastion"
+ssh ${SSH_OPTS} "${BASTION_USER}@${BASTION_IP}" -i "${WORKDIR}/temp_ssh_key" \
+  "oc --kubeconfig='${HUB_KUBECONFIG_PATH}' \
+    get secret ${SPOKE_CLUSTER}-admin-kubeconfig \
+    -n ${SPOKE_CLUSTER} \
+    -o jsonpath='{.data.kubeconfig}' | base64 -d > '${SPOKE_KUBECONFIG}'"
+
+echo "Wait for spoke worker node to be Ready"
+ssh ${SSH_OPTS} "${BASTION_USER}@${BASTION_IP}" -i "${WORKDIR}/temp_ssh_key" \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)